toolkit/tools/pkg/imagecustomizerlib/customizebootloader.go

```diff
@@ -34,12 +36,15 @@ func handleBootLoader(baseConfigPath string, config *imagecustomizerapi.Config,
 	return nil
 }
 
-func hardResetBootLoader(baseConfigPath string, config *imagecustomizerapi.Config, imageConnection *ImageConnection,
+func hardResetBootLoader(ctx context.Context, baseConfigPath string, config *imagecustomizerapi.Config, imageConnection *ImageConnection,
 	partUuidToFstabEntry map[string]diskutils.FstabEntry, newImage bool,
 ) error {
 	var err error
 	logger.Log.Infof("Hard reset bootloader config")
 
+	_, span := otel.GetTracerProvider().Tracer(OtelTracerName).Start(ctx, "reset_bootloader")
```

Comment:
In the future we will likely add support for a soft reset. So, it would be good to call this `hard_reset_bootloader`.